packages/xo-server/src/xo-mixins/backups.js

```diff
@@ -305,19 +306,41 @@ export default class {
   }
 
   async listRemoteBackups(remoteId) {
```

Comment:
This method is only used for legacy backups which do not support proxies, no need to update it.